CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +10,10 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+                Game gameObject = new Game();
+                gameObject.DiceRoll();
+                Testing testing = new Testing();
+                testing.DebugProgram();
         }
     }
 }
```

Comment:
Code follows C# style conventions. Functions well.

Reply:
Functions run well and good flow of code